kateryna-tolochko/README.md

```diff
@@ -0,0 +1,91 @@
+# Getting started with AliExpress
+
+AliExpress is a popular online marketplace where you can find a wide variety of products at affordable prices. Whatever you are looking for, AliExpress offers a huge selection from sellers around the world. If you are new to AliExpress, this guide will help you get started and make the most of your shopping experience.
+
+## Step 1. Create an account
+
+1. Visit the [AliExpress website](https://www.aliexpress.com/).
+
+2. Hover over **Account** on the top right to open a drop-down menu and click **Register**.
+
+!['Register' button in 'Account' menu](https://i.ibb.co/YpHZKXc/screen.png)
+
+3. After clicking, you should see a registration form and different options for signing up through social media accounts. Select your preferred method.
+
+   3a. If you choose to sign up through your social media accounts, you will be redirected to a page where you will need to give AliExpress permission to access your account information. Follow the on-screen instructions to complete the process.
```

Comment:
Increase indents for 3a and 3b (if possible, always check it on the preview to ensure it displays properly).